Reneiski_Homework1/ReneiskiSelenium11/Tests/Tests.cs

```diff
@@ -0,0 +1,121 @@
+﻿using NUnit.Framework;
+using ReneiskiSelenium11.PageObjects.ButtonPage;
+using ReneiskiSelenium11.PageObjects.CheckBoxPage;
+using ReneiskiSelenium11.PageObjects.LinksPage;
+using ReneiskiSelenium11.PageObjects.RadioButtonPage;
+using ReneiskiSelenium11.PageObjects.WebTablePage;
+using ReneiskiSelenium11.PageObjects.WebTablePage.RegistrationForm;
+
+namespace ReneiskiSelenium11.Tests
+{
+    public class Selenium : BaseTest
+    {
+
+        [Test]
+        public void FirstCheckBoxTest()
+        {
+            CheckBoxPage check = new CheckBoxPage(_driver);
+
+            check.GoToPage("checkbox");
```

Comment:
you may create OpenCheckboxPage inside CheckBox page that will go to a specific URL + return a new instance of the CheckboxPage

Reply:
Fixed